blog/content/post/2020-04-27-trenchboot-nlnet-release-04.md

```diff
@@ -0,0 +1,290 @@
+      02` data under 0xd000 address.
+
+      3. If above data is the same as in data structure in [landing-zone source
+      code](https://github.com/TrenchBoot/landing-zone/blob/master/include/boot.h#L50)
+      then it is **bootloader information header**. In given example, it is
+      placed after code section in LZ.
+
+      Debug version of landing zone can be checked exactly the same way. Just
+      make sure, you read address properly, as it probably isn't the same
+      value as in non-debug landing-zone.
+
+      1. Check value of second word of `lz_header_debug.bin` file using
+      `hexdump` tool.
+
+      ```
+      $ hexdump -C lz_header_debug.bin | head                                              
+      00000000  d4 01 00 e0 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+      00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+      *
+      000001d0  00 00 00 00 89 c5 8d a5  00 02 00 00 b9 14 01 01  |................|
+      000001e0  c0 0f 32 83 e0 f9 0f 30  01 ad ac 2b 00 00 01 ad  |..2....0...+....|
+      000001f0  52 02 00 00 01 ad 00 a0  00 00 01 ad 00 90 00 00  |R...............|
+      00000200  01 ad 08 90 00 00 01 ad  10 90 00 00 01 ad 18 90  |................|
+      00000210  00 00 01 ad 00 50 00 00  0f 01 95 aa 2b 00 00 b8  |.....P......+...|
+      00000220  10 00 00 00 8e d8 8e c0  0f 20 e1 83 c9 20 0f 22  |......... ... ."|
+      00000230  e1 8d 85 00 a0 00 00 0f  22 d8 b9 80 00 00 c0 0f  |........".......|
+      ```
+
+      > Value of the second word (`00 e0`) is size of code section and
+      concurrently offset (address) of bootloader information header. Used
+      notation is little-endian so the offset is actually `0xe000` (NOT 0x00e0).
+
+      2. Check the content of `lz_header_debug.bin` file from `0xe000` address.
+
+      ```
+      $ hexdump -C -s 0xe000 lz_header_debug.bin | head                                    
+      0000e000  78 f1 26 8e 04 92 11 e9  83 2a c8 5b 76 c4 cc 02  |x.&......*.[v...|
+      0000e010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+      *
+      0000e030  04 00 00 00 10 00 00 00  05 00 00 00 47 4e 55 00  |............GNU.|
+      0000e040  01 00 00 c0 04 00 00 00  01 00 00 00 00 00 00 00  |................|
+      0000e050
+      ```
+
+      > As you can see, there is `78 f1 26 8e 04 92 11 e9  83 2a c8 5b 76 c4 cc
+      02` data under 0xe000 address.
+
+      3. If above data is the same as in data structure in
+      [landing-zone source code](https://github.com/TrenchBoot/landing-zone/blob/master/include/boot.h#L50)
+      then it is **bootloader information header**. In given example, it is
+      placed after code section in LZ.
+
+## Enable DRTM in custom Linux built
+
+Our custom Linux built is called **meta-trenchboot**. To use it on your PC
+Engines apu2 platform, all you need to have is:
+
+      - SSD disk to store image
```

Comment:
Does it have to be SSD? Would USB also be good?

Reply:
That is information from @Asiderr. Please verify it to be 100% sure.